multstatelike_test.go

```diff
@@ -18,10 +18,11 @@ func TestPCalcLikePatternsMS(t *testing.T) {
 	for i := range modelparams {
 		modelparams[i] = 1.0
 	}
-	x := gophy.NewMultStateModel()
+	x := gophy.NewModel()
+	x.Alph = "mult"
```

Comment:
I think this would be better not as a string like "mult". it may introduce errors like "MULT" "multi", etc. so i think it should be a struct of const perhaps? kinda like an enum but go doesn't do that atm